packages/gestalt-datepicker/src/DatePicker.jsdom.test.js

```diff
@@ -105,4 +105,29 @@ describe('DatePicker', () => {
       'react-datepicker__day react-datepicker__day--013 react-datepicker__day--selected',
     );
   });
+
+  test('accepts utc timezone dates', () => {
+    const initialUTCDate = new Date(Date.UTC(2018, 2, 24));
```

Comment:
can we refine this test to get a non-UTC date and expect a UTC to be displayed instead. It should show a change in day date, something like 


`    const initialUTCDate = new Date((2018, 2, 24)); // You might need to set time maybe?
`


`    fireEvent.focus(screen.getByDisplayValue('03/25/2018')); // we could see the effect taking place
`

Reply:
Could also test `getDisplayDate`
`getUpdatedDate` using  `hasBeenCalledWith`

you might have to mock them. 

This test might be a little more tricky, I can help with it. 

Timezones are really important so we want to make sure the testing and functionality is robust.